### PR TITLE
Added flag: --disable-pb and --yes/-y

### PR DESCRIPTION
**Pull request by sanyamsinghal**
1. --disable-pb: it is for disabling progress bar during import data
2. --assumeyes or -y: it is to assume yes for all the questions asked during migration steps.

### CODE REVIEW

File: yb_migrate/cmd/import.go [hunk 1]
```diff
@@ -143,6 +143,9 @@ func registerCommonImportFlags(cmd *cobra.Command) {
 
 	cmd.Flags().BoolVar(&loadBalancerUsed, "load-balancer", false,
 		"true - if given --target-db-host is a load balancer ip (default false)")
+
+	cmd.PersistentFlags().BoolVar(&disablePb, "disable-pb", false,
+		"number of Parallel Jobs to extract data from source database")
```
Comment (amit-yb) [inline yb_migrate/cmd/import.go:148]: Fix the help string.

File: yb_migrate/cmd/root.go [hunk 2]
```diff
@@ -72,6 +74,9 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVarP(&exportDir, "export-dir", "e", "",
 		"export directory (default is current working directory")
+
+	rootCmd.PersistentFlags().BoolVarP(&askPromptResp, "assumeyes", "y", false,
```
Comment (amit-yb) [inline yb_migrate/cmd/root.go:78]: Name the command as `--yes`.
Rename the variable `askPromptResp` as `doNotPrompt` (note that the meaning is negated.)

Comment (amit-yb) [reply yb_migrate/cmd/root.go:78]: Will it be possible to populate the option value directly in the `&utils.DoNotPrompt` ? With that you won't need the `initCommonGlobalVars()` section.

File: yb_migrate/cmd/import.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb_migrate/cmd/import.go:148]: ohh!! thanks

**Review by amit-yb — APPROVED** [review]
LGTM. Thanks for quickly making the change.

Will it be possible for you to add `--yes` and `--disable-pb` options appropriately in the `migtests/scripts/functions.sh`?

**Comment by sanyamsinghal** [comment]
> LGTM. Thanks for quickly making the change.
> 
> Will it be possible for you to add `--yes` and `--disable-pb` options appropriately in the `migtests/scripts/functions.sh`?

Sure, can do that as a part of separate PR

**Pull request merged**